components/Tabs/src/TabBarView/MDCTabBarView.m

```diff
@@ -311,8 +314,15 @@ - (void)layoutSubviews {
   CGFloat availableWidth = CGRectGetWidth(self.bounds);
   CGFloat requiredWidth = [self justifiedWidth];
   BOOL canBeJustified = availableWidth >= requiredWidth;
-  self.containerView.distribution = canBeJustified ? UIStackViewDistributionFillEqually
-                                                   : UIStackViewDistributionFillProportionally;
+  if (canBeJustified) {
```

Comment:
Please flip for RTL here based on `effectiveUserInterfaceLayoutDirection`

Reply:
Done